crates/rolldown/src/hmr/hmr_ast_finalizer.rs

```diff
@@ -31,8 +31,25 @@ pub struct HmrAstFinalizer<'me, 'ast> {
   pub module: &'me NormalModule,
   pub affected_module_idx_to_init_fn_name: &'me FxHashMap<ModuleIdx, String>,
   pub use_pife_for_module_wrappers: bool,
-  //Internal state
-  pub import_binding: FxHashMap<SymbolId, String>,
+
+  // --- Internal state
+  /// For
+  ///
+  /// ```js
+  /// import foo, { bar } from './foo.js';
+  /// ```
+  ///
+  /// , assuming `foo` and `bar` have symbol id `1` and `2`, we will store mapping like:
+  ///
+  /// - 1 => "import_foo_1"
```

Comment:
Should this be `import_foo_1.default`? or Should the code above be `import * as foo from './foo.js'; import { bar } from './foo.js'`?
Or is the value for default export and namespace export the same?

Reply:
Yeah. It should be default, wrong comment.